Spigot-Server-Patches/0521-Allow-delegation-to-vanilla-chunk-gen.patch

```diff
@@ -0,0 +1,88 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: MiniDigger <admin@minidigger.me>
+Date: Wed, 29 Apr 2020 02:10:32 +0200
+Subject: [PATCH] Allow delegation to vanilla chunk gen
+
+
+diff --git a/src/main/java/net/minecraft/server/ChunkConverter.java b/src/main/java/net/minecraft/server/ChunkConverter.java
+index 8dd940fd94e4d129d1b8e20a91310028477bbd9b..24c64e5f487ef8ccd508071feb88712216c7b66d 100644
+--- a/src/main/java/net/minecraft/server/ChunkConverter.java
++++ b/src/main/java/net/minecraft/server/ChunkConverter.java
+@@ -17,7 +17,7 @@ import org.apache.logging.log4j.Logger;
+ public class ChunkConverter {
+ 
+     private static final Logger LOGGER = LogManager.getLogger();
+-    public static final ChunkConverter a = new ChunkConverter();
++    public static final ChunkConverter a = new ChunkConverter(); public static ChunkConverter getEmptyConverter() { return a; } // Paper - obfhelper
+     private static final EnumDirection8[] c = EnumDirection8.values();
+     private final EnumSet<EnumDirection8> d;
+     private final int[][] e;
+@@ -343,7 +343,7 @@ public class ChunkConverter {
+                         if ((Integer) iblockdata.get(BlockProperties.ah) >= j) {
+                             generatoraccess.setTypeAndData(blockposition, (IBlockData) iblockdata.set(BlockProperties.ah, j), 18);
+                             if (i != 7) {
+-                                EnumDirection[] aenumdirection = null.f;
++                                EnumDirection[] aenumdirection = f; // Paper - decomp fix
```

Comment:
Should this go in 0003-MC-Dev-fixes.patch?

Reply:
I actually have no clue, I have seen decomp fixes in other patches before, I can move it if wanted.